package.json

```diff
@@ -33,8 +33,9 @@
   "scripts": {
     "reskindex": "node scripts/reskindex.js -h header",
     "reskindex:watch": "node scripts/reskindex.js -h header -w",
-    "build": "npm run reskindex && babel src -d lib --source-maps",
-    "build:watch": "babel src -w -d lib --source-maps",
+    "build": "npm run emoji-data-strip && npm run reskindex && babel src -d lib --source-maps",
```

Comment:
I think you'll need to create the lib directory somewhere, as babel was doing it for us before.